src/rpc/mining.cpp

```diff
@@ -252,7 +252,7 @@ static UniValue generatetoaddress(const JSONRPCRequest& request)
                 RPCExamples{
             "\nGenerate 11 blocks to myaddress\n"
             + HelpExampleCli("generatetoaddress", "11 \"myaddress\"")
-            + "If you are running the bitcoin core wallet, you can get a new address to send the newly generated bitcoin to with:\n"
+            + "If you are using the " PACKAGE_NAME " wallet, you can get a new address to send the newly generated bitcoin to with:\n"
```

Comment:
nit: either remove the colon, or add one at line 253 s/myaddress/myaddress:/

Looking at the other helps like listwallets, generatetodescriptor, etc., no colons seem to be used.

Reply:
RPCMan can take care of all formatting needs in the future. Instead of playing formatting ping-pong, I recommend to make progress on #18531